src/components/Login/index.js

```diff
@@ -10,8 +10,56 @@ const LoginWrapper = styled.div`
   margin: auto;
   display: flex;
   justify-content: center;
+  font-family: Arial;
+  color: #0085ff;
 `;
 
+const PageWrapper = styled.div`
+  background-color: #eee;
+
+`;
+
+const SlingWrapper = styled.div`
+  margin:auto;
+  font-family: Arial;
+  display: flex;
+  justify-content: center;
+    color: #646464;
+`;
+
+
+const buttonStyle = {
```

Comment:
You should stick to one type of styles,
this can be converted to styled component by defining
```javascript
const Button = styled.button`
  background-color: #0085ff;
  width: 250px;
  border: none;
  margin: 5px;
  padding: 5px;
  color: white;
`;
```
We always start a styled component name with an uppercase letter since most components start with upper case letters.